default/stringtables/en.txt

```diff
@@ -4527,7 +4526,7 @@ EMPIRE_METERS
 Meters:
 
 RESEARCHED_TECHS
-Researched Techs: 
+'''Researched Techs: '''
```

Comment:
Just to make it clear there is a space at the end?

Reply:
Yes. As it has been done for EMPIRE_CAPITAL a few lines before.

It seems this space is unnecessary though, because AFAIK, "Researched Techs:" is only used in an empire's pedia article, and there is a break line after.

But, as I wasn't totally sure, I didn't remove this space, and add the 3 quotes to make clear the space at the end.